packages/mdc-icon-button/_variables.scss

```diff
@@ -20,4 +20,18 @@
 // THE SOFTWARE.
 //
 
-$mdc-icon-button-size: 24px !default;
+@import "@material/density/variables";
+
+$mdc-icon-size: 24px !default;
```

Comment:
All variable name should prefix with component name.

`$mdc-icon-button-icon-size`

Reply:
Done!